integrationTests/relayers/slowTests/semi-integration_test.go

```diff
@@ -0,0 +1,144 @@
+//go:build slow
+
+package slowTests
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"math/big"
+	"os"
+	"path"
+	"testing"
+
+	bridgeCore "github.com/multiversx/mx-bridge-eth-go/core"
+	"github.com/multiversx/mx-bridge-eth-go/integrationTests/relayers/slowTests/framework"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	numRelayers = 3
+	numOracles  = 3
+	quorum      = "03"
+	scAddress   = "erd1qqqqqqqqqqqqqpgqvc7gdl0p4s97guh498wgz75k8sav6sjfjlwqh679jy"
+)
+
+type BridgeProxyTestSetup struct {
```

Comment:
missing comments on all exported elements.
Why not move this new setup in a dedicated file in the `framework` package?